paper/draft.md

```diff
@@ -0,0 +1,28 @@
+> Here we will put the text for the paper. Later, it will be slightly reformated to meet JOSS's submission criteria. For now, it's mostly copy-paste-edit from the manual or readme.
+
+
+## Schema Salad: A bridge between document and record oriented data modeling and the Semantic Web
+
+### Summary
+Salad is a schema language for describing structured linked data documents in JSON or YAML documents. A Salad schema provides rules for preprocessing, structural validation, and link checking for documents described by a Salad schema. Salad builds on JSON-LD and the Apache Avro data serialization system and extends Avro with features for rich data modeling such as inheritance, template specialization, object identifiers, and object references. Salad was developed to provide a bridge between the record oriented data modeling supported by Apache Avro and the Semantic Web.
```

Comment:
Just some thougths. Assuming I'm a user who read it for the first time. When I see the sentence
> Salad was developed to provide a bridge between the record-oriented data modeling supported by Apache Avro and the Semantic Web

I have a question "Why do we need that bridge? How exactly it makes my life easier? Where do I use record-oriented data?"

Reply:
In my opinion the README is not great. It does not explain in layman's terms why we have schema salad and why we need it.

Reply:
You can argue the README is for experts. But I think the JOSS paper is an opportunity to explain things well.